src/seedu/addressbook/ui/MainWindow.java

```diff
@@ -186,14 +192,14 @@ void onCommand(ActionEvent event) {
         } catch (Exception e) {
             displayStatus(e.getMessage() + "\nExiting App...");
             Main.LOGGER.log(Level.WARNING, e.getMessage());
-            // If this error occurs, data between addressbook and storage is likely to be desynced,
+            // If this error occurs, data between the app and storage is likely to be desynced,
             // so we force close the app to ensure synchronisation.
             final int delayTime = 3000;
             exitApp(delayTime);
         }
     }
 
-    /** Exits the app after a given delay*/
+    /** Exits the app after a set delay of 500ms*/
```

Comment:
Seems like a pretty lame comment though, considering that it needs to be changed if we change the delay, which makes it a "bad comment"

Reply:
its a magic number :)

Reply:
Its not, its under a named variable

Reply:
I'll remove the 500 then. I need to update my code too.